FoodDeliveryUI/FoodDeliveryUI/FoodDeliveryUI.Shared/MainPage.xaml

```diff
@@ -0,0 +1,80 @@
+﻿<Page
+    x:Class="FoodDeliveryUI.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:FoodDeliveryUI" 
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"   
+    Background="#09153c">
+    <!-- Page layout structure-->
+    <Grid Padding="30,50,30,0" ColumnSpacing="20" RowDefinitions="Auto,Auto,Auto,Auto,Auto,Auto,Auto" ColumnDefinitions="*,*,Auto">
+
+        <!-- 1. Profile picture-->
+
+        <!-- Profile image-->
+        <Image Grid.Column="0" Grid.Row="0" HorizontalAlignment="Left" Source="marie" Height="50" Width="50" />
+        <!-- Search and Book Buttons-->
+        <Button Grid.Column="1" Grid.Row="0" Background="#1a204e" Height="50" Width="50" CornerRadius="25" HorizontalAlignment="Right">
+            <Image Source="find"/>
+        </Button>
+        <!-- Location Button-->
+        <Button Grid.Column="2" Grid.Row="0" Background="#1a204e" Height="50" Width="50" CornerRadius="25" HorizontalAlignment="Right">
+            <Image Source="book"/>
+        </Button>
+
+        <!-- 2. Menu to Eat-->
+
+        <!-- Gretting label -->
+        <TextBlock Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="1" Text="Hello, Rachel!" FontSize="30" Foreground="White" FontWeight="Bold" Margin="0,20,0,0" />
+        <!-- Welcome Label-->
+        <TextBlock Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="2" Text="Wanna eat tonight?" FontSize="15" Foreground="White"/>
+        <!-- Menu list-->
+        <ListView Grid.Row="3" Grid.Column="0" Grid.ColumnSpan="3" Margin="0,20,0,0" ItemsSource="{x:Bind food}">
+            <ListView.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <StackPanel Orientation="Horizontal"/>
+                </ItemsPanelTemplate>
+            </ListView.ItemsPanel>
+            <ListView.ItemTemplate>
+                <DataTemplate x:DataType="local:Food">
+                    <Frame CornerRadius="15" BorderBrush="#1a204e" BorderThickness="1" Height="200" Width="150" HorizontalAlignment="Left" Margin="-5,0" >

```

Comment:
`Frame` in uwp/winui/uno is a control used to display pages with built-in navigation support. This is probably copy-paste error when porting from other languages. a `Border` would be more appropriate here.

Reply:
additionally, using a negative left-right-margin is not the best way to adjust the spacing here.
and, the selection looks a bit off:
![image](https://user-images.githubusercontent.com/2359550/201985585-b0e398a4-a4a3-48f6-9169-feca3398b1bf.png)

you should instead use ItemContainerStyle to override the default padding of `12,0` it to a more desired value:
![image](https://user-images.githubusercontent.com/2359550/201985936-7c04c920-e989-4576-9b6d-e0ecc7f85dbb.png)
```xml
<ListView.ItemContainerStyle>
	<Style TargetType="ListViewItem">
		<Setter Property="Padding" Value="10" />
	</Style>
</ListView.ItemContainerStyle>
```

or, even considering moving the "border" part into ItemContainerStyle, so the selection highlight line up to the border:
![image](https://user-images.githubusercontent.com/2359550/201986636-ca1f6efc-9476-4738-8c4b-09e37ef4732f.png)
```xml
<ListView.ItemContainerStyle>
	<Style TargetType="ListViewItem">
		<Setter Property="BorderThickness" Value="1" />
		<Setter Property="BorderBrush" Value="#1a204e" />
		<Setter Property="CornerRadius" Value="15" />
		<Setter Property="Margin" Value="10" />
		<Setter Property="Padding" Value="0" />
		<Setter Property="Height" Value="200" />
		<Setter Property="Width" Value="150" />
	</Style>
</ListView.ItemContainerStyle>
<ListView.ItemTemplate>
    <DataTemplate x:DataType="local:Food">
        <!--<Border CornerRadius="15" BorderBrush="#1a204e" BorderThickness="1" Height="200" Width="150" HorizontalAlignment="Left" >-->
            <Grid Padding="10" RowDefinitions="*,Auto,Auto">
                <Image Grid.Row="0" Source="{x:Bind Picture}" Height="100" Width="100"/>
                <TextBlock Grid.Row="1" Text="{x:Bind Name}" FontWeight="Bold" Foreground="White" HorizontalAlignment="Center"/>
                <TextBlock Grid.Row="2" Text="{x:Bind Description}" Foreground="White" HorizontalAlignment="Center"/>
            </Grid>
        <!--</Border>-->
    </DataTemplate>
</ListView.ItemTemplate>
```